modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -54,21 +52,20 @@ The following filtering options are available:
 ** **Down**. The target is currently down and not being scraped for metrics.
 
 * **Source** filters:
-** **Platform**. Platform-level targets relate only to default Red Hat OpenShift Service on AWS projects. These projects provide core Red Hat OpenShift Service on AWS functionality.
+** **Platform**. Platform-level targets relate only to default {product-rosa} projects. These projects provide core {product-rosa} functionality.
```

Comment:
Just checking if a ROSA reference needs to be conditionally included or if it's fine as-is

Reply:
We have it in the common attributes so it made sense for me to conditionize it. Let me know if you think otherwise, thanks!